ops/cloud-deployment/backend-api-deploy.bicep

```diff
@@ -179,6 +179,17 @@ resource apiFunctionApp 'Microsoft.Web/sites@2023-12-01' = {
     properties: prodFunctionAppConfigProperties
   }
 
+  resource slotConfigNames 'config' = {
+    name: 'slotConfigNames'
+    properties: {
+      appSettingNames: [
+        'AzureWebJobsStorage'
```

Comment:
The values for this are actually not different between the slots so I'm not sure it needs to be listed, but currently is set up this way.

Reply:
Actually, they should be different.